cpp/src/neighbors/cagra_c.cpp

```diff
@@ -426,23 +426,23 @@ extern "C" cuvsError_t cuvsCagraIndexDestroy(cuvsCagraIndex_t index_c_ptr)
   });
 }
 
-extern "C" cuvsError_t cuvsCagraIndexGetDims(cuvsCagraIndex_t index, int* dim)
+extern "C" cuvsError_t cuvsCagraIndexGetDims(cuvsCagraIndex_t index, int64_t* dim)
 {
   return cuvs::core::translate_exceptions([=] {
     auto index_ptr = reinterpret_cast<cuvs::neighbors::cagra::index<float, uint32_t>*>(index->addr);
```

Comment:
Tangential question:  How come `index->addr` is a `uintprt_t` and not a `void*`?

(Functionally, it's likely equivalent, except for the ability to use `static_cast` instead of `reinterpret_cast`.)